app/routes/routes_helper.py

```diff
@@ -0,0 +1,12 @@
+from flask import abort, make_response
+
+def validate_model(cls, model_id):
```

Comment:
Nice helper, having a validation function for the request body might also be good. 

You could have the function take in a list of required fields and verify that the request body has them could be very useful.